src/helperFunctions/headerFetchRequests.js

```diff
@@ -0,0 +1,5 @@
+export let fetchSetList = () => {
+    return (fetch('https://cors-anywhere.herokuapp.com/https://phantom-closet.herokuapp.com//public/sets', 
```

Comment:
maybe we should put this in the .env file? I still mostly use local host for experimenting and that way we won't have a git diff every-time one of us pushes a commit:-p I can do it on my next commit too.